px-data-grid.html

```diff
@@ -1130,6 +1149,20 @@
 
           this._vaadinGrid.clearCache();
         }
+
+        _autoHeightChanged(autoHeight) {
+          if (!this._vaadinGrid) {
+            return;
+          }
+          if (autoHeight) {
```

Comment:
Instead of this we can do something like so
```html
<vaadin-grid auto-height$="[[autoHeight]]"></vaadin-grid>
```

Reply:
This will add and remove attribute from vaadin-grid

Reply:
This might break if vaadin-grid actually starts to use attribute named like that.